### ISSUE TITLE
Getting non-mouse PRO ids into Noctua for use in annotation

### ISSUE THREAD

**Issue by hdrabkin** [geneontology/noctua#429]
All of PRO IDS should be available in Noctua (ids for human, pombe, etc); can these be loaded from PRO directly (could (PRO to supply a GPI file).

**Comment by kltm** [comment]
So the action here would be to get a GPI file that covered PRO (one way or another) and include it in NEO, yes?

**Comment by hdrabkin** [comment]
Essentially. Right now, MGI handles the mouse specific, but PRO has lots of human, Pombe, etc. that aren't being submitted, so there is no way to use them in Noctua.

**Comment by cmungall** [comment]
The translation is simple.

However there are things here that require discussion with the larger group

 - [ ] size issues and hardware. All of PRO is large (300k classes), we need to determine if we will need to up memory for minerva or solr. Also we don't have any guaranteed estimated upper bounds like with most ontologies. In theory PRO could grow orders of magnitude over the years.
 - [ ] size issues for usability. Has the potential to flood autocomplete results making it harder to find what is needed. 
 - [ ] orthogonality issues. We will have 'duplicate' entries for GCRPs, which can potentially confuse curators and bring up a host of SOP issues. One option is to exclude GCRPs from PRO and only use PRO for subclasses of GCRPs, but this is awkward

**Comment by kltm** [comment]
As these issues still seem to be open, is this still on the table for Corvallis?

**Comment by hdrabkin** [comment]
Yes;

**Comment by kltm** [comment]
To clarify: is the intention here to have all of this fixed and being used before Corvallis?

**Comment by cmungall** [comment]
No, nothing should be changed or implemented before we discuss and agree 
upon my earlier comments, which is best discussed at Corvalis

**Comment by hdrabkin** [comment]
Since I assume PRO might be the one to supply a PRO specific GPI, I very much doubt this could be done by then!  Darren Natale at PRO (who will be at the meeting) would need guidance at the least.

**Comment by cmungall** [comment]
No, that part is trivial / not necessary. We actually convert GPI to obo/owl. PRO is already in obo/owl, and could be slotted in naturally. However, we can't do that until the issues I mention above are discussed.

**Comment by cmungall** [comment]
Removing from current milestones. Will be discussed when I attend PRO meeting
